### PR TITLE
As a user, I want to be able to view and leave comments about dishes and restaurants.

### PR DESCRIPTION
**Pull request by danidanone17**
Description:
Patat does not allow any user feedback regarding the data provided.

Checklist view comments
- [x] database changes
- [x] everybody can view comments
- [x] order comments by date; most recent comment first
- [x] comments go along with username, date
- [x] comments should be limited to 5000 characters
- [x] display comments in restaurant view below restaurand/dish but collapsed by default 

Checklist add comments
- [x] comments can only be written if user (which is a customer) is logged in
- [x] if user is logged in, there is a textbox below existing comments to a restaurant/dish to create a comment
- [x] implement a new control that supports character count and use it for creating comments
- [x] if a dish is deleted, its comments are deleted as well
- [x] make view of comments prettier (e.g. make it visible for the user that 'Toggle comments' can be clicked on)
- [x] a user can add muliple comments to the same dish
- [x] comment description is too long
- [x] reload view, after comment has added to dish/restaurant
- [ ] validate comment description before save

@anna-averianova Almost done, except from validation of description, which I need help with. Can we take a look at that tomorrow? The rest is ready for review.

### CODE REVIEW

File: docs/131111-categories.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Create table Comments
```
Comment (anna-averianova) [inline docs/131111-categories.sql]: Change the name of this file ;)

File: app/javascripts/views/jquery.imbiss-index.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
 $('#map-small').patatmap({
     draw: 'panorama'
-});
\ No newline at end of file
+});
```
Comment (anna-averianova) [inline app/javascripts/views/jquery.imbiss-index.js:3]: :+1:

File: app/models/Comments.php [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+            ->orderBy('Comments.creation_date DESC')
+            ->all();
+
+        return $comments;
+    }
+
+    /**
+     * Gets comments for a dish determined by the given id
+     * @param $id dish id
+     * @return array comments for a dish determines by the given id
```
Comment (anna-averianova) [inline app/models/Comments.php:132]: Very important flaw: you have a typo in "determines"

File: docs/131111-comments.sql [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Create table Comments
```
Comment (anna-averianova) [inline docs/131111-comments.sql:1]: Which of the sql is the correct one?

**Comment by anna-averianova** [comment]
@danidanone17 I can't run this as I get PHP Fatal error:  Call to undefined method controllers\Dishes::setRenderArg() in /Applications/MAMP/htdocs/lmvc-patat/app/controllers/Dishes.php on line 21
How did you get around this one?

Otherwise don't know what to complain about, only two sql files maybe (remove one?)

**Comment by tdeekens** [comment]
@SEP007/coders If you want somebody to review your code. Its a good idea to merge in `master` before and check if everything works. This is justified by the fact that our dependencies are not managed Patat's git as submodules but by composer.

To be more illustrative in this case @anna is running on the latest sprints changed in our modules. Good stuff. Last sprint we changed the whole rendering process placing it in a separate module called `rendering`. @dani on the other hand still seems to have partially old dependencies `lmvc-modules` and built her controller without using the latest sprint's changed. No fingers to cross there, it just won't work.

Hence update the dependencies e.g by just doing a `git pull master` in `lmvc`, `lmvc-utils` and `lmvc-modules` as lil boi is only allowed to mess with those and no User Stories.

I'll be kind after I merged my stuff and fix this branch. So anya can review. I'll ping back inabit.

**Comment by tdeekens** [comment]
Merged `master` in with conflicts but resolved them.

@anna-averianova should work for you now, for me it does.

@danidanone17 please do a `php composer.phar update` otherwise you run into trouble. If you do anyways we'll revolve them tomorrow.

**Comment by anna-averianova** [comment]
@danidanone17 sorry, disregard my stupid comment about sql

**Comment by anna-averianova** [comment]
@tdeekens thanks!!111 ! :trophy: :trophy: :trophy: :trophy: :trophy: :trophy: :trophy:

**Comment by anna-averianova** [comment]
@danidanone17 except for German buttons — so far so good.
Only have issues related to UI, but I guess that is highly subjective so we shall discuss it altogether.

**Pull request merged**